packages/parser/lib/css_lexer.re

```diff
@@ -2,101 +2,114 @@
   * Reference:
   * https://www.w3.org/TR/css-syntax-3/ */
 module Sedlexing = Lex_buffer;
-module Parser = Css_parser;
+// open Sedlexing;
+// module Parser = Css_parser;
 module Types = Css_types;
+module Tokens = Tokens;
+open Tokens;
```

Comment:
I prefer to not have an open, and annotate all `Tokens.`

Reply:
Agreed!